infra/src/main/scala/ch/epfl/scala/services/github/GithubModel.scala

```diff
@@ -0,0 +1,131 @@
+package ch.epfl.scala.services.github
+
+import cats.implicits.toTraverseOps
+import ch.epfl.scala.index.model.misc.GithubContributor
+import ch.epfl.scala.index.model.misc.GithubIssue
+import ch.epfl.scala.index.model.misc.Url
+import io.circe._
+import io.circe.generic.semiauto._
+
+object GithubModel {
+
+  case class Repository(
+      name: String, // cat
+      owner: String, // owner.login
+      avatartUrl: String, // owner.avatar_url
+      isPrivate: Boolean,
+      description: Option[String],
+      isFork: Boolean,
+      createdAt: String, // format: "2015-01-28T20:26:48Z",
+      updatedAt: String,
+      homepage: Option[String], // http://typelevel.org/cats/
+      stargazers_count: Int, // stars
+      forks_count: Int,
+      mirror_url: Option[String], // "mirror_url": "git://git.apache.org/spark.git",
+      forks: Int,
+      open_issues: Int,
+      default_branch: String, // master
+      subscribers_count: Int, // Watch
+      topics: Seq[String]
+  )
+
+  implicit val repositoryDecoder: Decoder[Repository] = new Decoder[Repository] {
```

Comment:
Maybe add a comment to explain why the `Decoder` cannot be derived automatically

Reply:
ok